src/main.js

```diff
@@ -136,85 +130,85 @@ btnBackMain.addEventListener('click', switchPageMain)
 btnShowMain.addEventListener('click', switchPage)
 btnSave.addEventListener('click', savePoster)
 
-
-function savePoster(){
-    posterInstance = new Poster(
-    posterImg.src,
-    posterTitle.innerText,
-    posterQuote.innerText
-  )
-  savedPosters.push(posterInstance)
-};
-
-
 btnShowPoster.addEventListener('click', function() {
-  event.preventDefault()
-  posterTitle.innerText = makeTitle.value
-  posterImg.src = makeImage.value
-  posterQuote.innerText = makeQuote.value
-
-  titles.push(posterTitle.innerText)
-  images.push(posterImg.src)
-  quotes.push(posterQuote.innerText)
-  switchPage()
-
+    event.preventDefault()
+    posterTitle.innerText = makeTitle.value
+    posterImg.src = makeImage.value
+    posterQuote.innerText = makeQuote.value
+
+    titles.push(posterTitle.innerText)
+    images.push(posterImg.src)
+    quotes.push(posterQuote.innerText)
+    switchPage()
 });
 
 // functions and event handlers go here 👇
 // (we've provided one for you to get you started)!
 randoImg = getRandomIndex(images)
 randoTitl = getRandomIndex(titles)
 randoQuote = getRandomIndex(quotes)
+
 posterTitle.innerText = titles[randoTitl]
 posterImg.src = images[randoImg]
 posterQuote.innerText = quotes[randoQuote]
```

Comment:
Is there a way you could move this into a function perhaps called `displayRandomPoster` that can be called upon page load? I would look into the other capabilities that addEventListener has to offer (it can do more than respond to a click - there are other page events that can trigger the event listener)

Reply:
Thank you so much. I will do that now.